crypto/async/arch/async_win.c

```diff
@@ -11,7 +11,7 @@
 #include "../async_locl.h"
 
 #ifdef ASYNC_WIN
-
+#define _WINSOCKAPI_  
```

Comment:
Indent the define to match the include below and reflect the nesting level of the conditional (all occurrences).
It would also be fine to put comments on the line before the define to indicate why they are present, but use C-style comments.